### DISCUSSION TITLE
development

### DISCUSSION THREAD

**Discussion by dhvcc** [Sora-reader/backend#57] [General]
Тут проще записывать обсуждения тикетов, кто что хочет взять и т.д.

**Comment by dhvcc** [comment]
@NikDark @zym43210 
Привет. Я сейчас начну только фронтом заниматься, так что бек будет на вас
Весь АПИ я уже описал и тикеты, по сути

Тогда, Никита возьмёт часть парсинга, а Юра - API

**Пожалуйста, не забывайте смотреть в [майлстоуны](https://github.com/Sora-reader/backend/milestones), где я описываю чего мы хотим и как это будет работать**

**Comment by NikDark** [comment]
@dhvcc 
Нужно будет обсудить этот коммент
https://github.com/Sora-reader/backend/issues/69#issuecomment-869430042

**Comment by dhvcc** [comment]
@MariaAfanasyeva @NikDark когда у вас будет время поговорить по поводу абстракции спайдеров и парсинг функций/классов? И вообще, есть у вас уже какие-то мысли/предложения?
